anesthetic/samples.py

```diff
@@ -389,13 +388,24 @@ class NestedSamples(MCMCSamples):
     beta: float
         thermodynamic temperature
 
+    logzero: float
+        The threshold for `log(0)` values assigned to rejected sample points.
+        Anything equal or below this value is set to `-numpy.inf`.
+        default: -numpy.inf
+
     """
 
     def __init__(self, *args, **kwargs):
         root = kwargs.pop('root', None)
+        logzero = kwargs.pop('logzero', -numpy.inf)
         if root is not None:
             reader = SampleReader(root)
             samples, logL, logL_birth = reader.samples()
+            logL = numpy.where(logL <= logzero, -numpy.inf, logL)
+            if not isinstance(logL_birth, int):
+                logL_birth = numpy.where(logL_birth <= logzero,
+                                         -numpy.inf,
+                                         logL_birth)
```

Comment:
Do we need the newline here?

Reply:
flake8 requires one new line, but not necessarily two, i.e. we could lift either `-numpy.inf` or `logL_birth` up by one line. Preference?

Reply:
I would have lifted `logL_birth`

Reply:
Done in [4faf61e](https://github.com/williamjameshandley/anesthetic/pull/71/commits/4faf61e4a4f600986f6dfbefaab81810b0accbbe).